packages/curve-ui-kit/src/shared/ui/Metric.tsx

```diff
@@ -212,7 +214,7 @@ const MetricValue = ({
             </Typography>
           )}
 
-          <Typography variant={fontVariant} color="textPrimary">
+          <Typography variant={fontVariant} color={fontColor}>
```

Comment:
I'd put the default value here with `fontColor ?? 'textPrimary`, as technically the parameter is still optional and can be `undefined`